styles/trip-house-styles.css

```diff
@@ -39,3 +40,58 @@
 	font-size: 24px;
 	color: #383838;
 }
+
+/*footer-section*/
+
+.footer {
+	height: 625px;
+	background-color: #10367c;
+}
+
+.link-section {
+	padding-top: 120px;
+	color: #ffffff;
+	font-size: 20px;
+	font-weight: 400;
+	margin: unset;
+	display: flex;
+}
+
+.link {
+	margin: unset;
+}
+
+.about-column {
```

Comment:
Replace next three classes with class, which i described you in html file (width:33.3%) and if you need some special properties, you can use pseudoclasses (last:child, nth-child, etc)